recruiter/parser/hackernews.py

```diff
@@ -1,5 +1,7 @@
-from base import *
+from recruiter.parser.base import *
 from recruiter.candidate import Candidate
+import six
```

Comment:
Yeah, that sounds reasonable.  Maybe I should just run a PEP-8 formatter over the code.  Thoughts?